contracts/script/deploy/7683/DeployT1ERC7683Pull.s.sol

```diff
@@ -39,13 +40,13 @@ contract DeployT1ERC7683Pull is DeploymentUtils {
         logAddress("L1_T1_PULL_BASED_7683_IMPLEMENTATION_ADDR", address(impl));
         logAddress("L1_T1_PULL_BASED_7683_PROXY_ADDR", address(proxy));
 
-        logEnd("DeployRouterPullBasedERC7683 to L1");
+        logEnd("DeployT1ERC7683Pull to L1");
     }
 
     function l1_init() external {
         uint256 deployerPk = vm.envUint("L1_DEPLOYER_PRIVATE_KEY");
         address L1_T1_7683_PROXY_ADDR = vm.envAddress("L1_T1_PULL_BASED_7683_PROXY_ADDR");
-        address L2_T1_7683_PROXY_ADDR = vm.envAddress("L2_T1_PULL_BASED_7683_PROXY_ADDR");
+        address L2_T1_7683_PROXY_ADDR = vm.envAddress("L1_L2_T1_PULL_BASED_7683_PROXY_ADDR");
```

Comment:
this var name is confusing to me. What does `L1_L2_T1` mean?

Reply:
It is an L1 contract with L2 as counterpart. I introduce this wording to differentiate the contracts vars from L1<>L2 to PR1<>L2. I know 𝚝𝟷 in the after this prefix is confusing. Can you think of a better alternative?

Reply:
we can just say the name of the network.

Reply:
That would change all the config var names.